### PR TITLE
[Term Entry] PyTorch Tensors .ones()

### PR DESCRIPTION
**Pull request by kerikek**
### Description

- Added the .ones() function
- File path:  docs/content/pytorch/concepts/tensors/terms/ones/ones.md
- This is my first entry, thanks for the opportunity to contribute. Please let me know if anything needs to be updated/changed/etc.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by kerikek** [comment]
Hello @avdhoottt @CLAassistant 

Screenshot 2024-07-31 at 1 23 22 PM

"Script Kitty" was the initial username I made when first setting up GitHub, but it is no longer my username. Is there any way to remove it from this CLA prompt, or do I need to resubmit everything from my current username?

File: content/pytorch/concepts/tensors/terms/ones/ones.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- `size`: This parameter is required and specifies tensor shape.
+- `out`: This optional parameter is the output tensor. Default: `None`
+- `dtype`: This optional parameter denotes the desired type for the returned tensor. Default: `None`
+- `layout`: This optional parameter designates the layout desired for the returned tensor. Default: `torch.strided`
+- `device`: This optional parameter specifies the desired device of the returned tensor (e.g. `torch.device('cpu')` or `torch.device('cuda')`). Default: `None`
+- `requires_grad`: This optional parameter determines if autograd should record the operations on the returned tensor. Default: `False`
+
+## Example
+
+The following example shows the use of the `.ones()` function.
```
Comment (avdhoottt) [inline content/pytorch/concepts/tensors/terms/ones/ones.md:42]:
```suggestion
The following example shows how to use the `.ones()` function:
```

File: content/pytorch/concepts/tensors/terms/ones/ones.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+The code above generates the following output:
+
+```
+tensor([1., 1., 1.])
+
+tensor([[1., 1., 1., 1., 1.],
+  [1., 1., 1., 1., 1.]])
+```
```
Comment (avdhoottt) [inline content/pytorch/concepts/tensors/terms/ones/ones.md:61]: The output should be inside a `shell` block.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Excellent work @kerikek!! The entry is almost correct and needs a few minor changes. I've suggested the change so please make them. Thanks!!

**Comment by avdhoottt** [comment]
> Hello @avdhoottt @CLAassistant
> 
> Screenshot 2024-07-31 at 1 23 22 PM
> "Script Kitty" was the initial username I made when first setting up GitHub, but it is no longer my username. Is there any way to remove it from this CLA prompt, or do I need to resubmit everything from my current username?

Hey @kerikek, you can fix it by adding the same email ID to the current account you used when committing the changes. Try it and let me know if it fixes the problem.

File: content/pytorch/concepts/tensors/terms/ones/ones.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  - 'Libraries'
+  - 'Machine Learning'  
+  - 'Neural Networks'
+  - 'Syntax'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+  - 'machine-learning'
+  - 'generative-ai-models-generating-data-using-generative-adversarial-networks'
+  - 'generative-ai-models-getting-started-with-autoencoders'
```
[the remark this answers is not in the document]
Comment (kerikek) [reply content/pytorch/concepts/tensors/terms/ones/ones.md]: Made changes based on your recommendations.

File: content/pytorch/concepts/tensors/terms/ones/ones.md (hunk 2 above)
Comment (kerikek) [reply content/pytorch/concepts/tensors/terms/ones/ones.md]: Made changes based on your recommendations.

**Comment by kerikek** [comment]
@avdhoottt I tried to add the email, but it's also from my previous private GitHub-generated password. It will not link to my account. Is there a way to go and change my commit history to my current email address?

**Comment by avdhoottt** [comment]
@kerikek Try using the `git-filter-repo` tool which can be used to alter commits. Use chatGPT or any other AI tool to validate this and understand the steps. If this doesn't work then unfortunately you'll need to create a separate PR again.

**Pull request closed without merging**